Ryujinx.Ava/Helper/LoggerAdapter.cs

```diff
@@ -0,0 +1,163 @@
+using Avalonia.Utilities;
+using System;
+using System.Text;
+
+namespace Ryujinx.Ava.UI.Helper
+{
+    using AvaLogger = Avalonia.Logging.Logger;
+    using AvaLogLevel = Avalonia.Logging.LogEventLevel;
+    using RyuLogger = Ryujinx.Common.Logging.Logger;
+    using RyuLogClass = Ryujinx.Common.Logging.LogClass;
+
+    public class LoggerAdapter : Avalonia.Logging.ILogSink
+    {
+        public static void Register()
+        {
+            AvaLogger.Sink = new LoggerAdapter();
+        }
+
+        private static RyuLogger.Log? GetLog(AvaLogLevel level) => level switch {
+            AvaLogLevel.Verbose => RyuLogger.Trace,
+            AvaLogLevel.Debug => RyuLogger.Debug,
+            AvaLogLevel.Information => RyuLogger.Info,
+            AvaLogLevel.Warning => RyuLogger.Warning,
+            AvaLogLevel.Error => RyuLogger.Error,
+            AvaLogLevel.Fatal => RyuLogger.Notice,
+            _ => throw new ArgumentOutOfRangeException(nameof(level), level, null)
+        };
+
+        public bool IsEnabled(AvaLogLevel level, string area)
+        {
+            return GetLog(level) != null;
+        }
+
+        public void Log(AvaLogLevel level, string area, object source, string messageTemplate)
+        {
+            GetLog(level)?.PrintMsg(RyuLogClass.Avalonia, Format<object, object, object>(area, messageTemplate, source, null));
+        }
+
+        public void Log<T0>(AvaLogLevel level, string area, object source, string messageTemplate, T0 propertyValue0)
+        {
+            GetLog(level)?.PrintMsg(RyuLogClass.Avalonia, Format<object, object, object>(area, messageTemplate, source, new object[] {propertyValue0}));
```

Comment:
```suggestion
            GetLog(level)?.PrintMsg(RyuLogClass.Avalonia, Format<object, object, object>(area, messageTemplate, source, new object[] { propertyValue0 }));
```
Same for the functions below.